src/Stratis.Features.FederatedPeg.Tests/Wallet/MultiSigTransactionsTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+using Stratis.Features.FederatedPeg.Wallet;
+using Xunit;
+
+namespace Stratis.Features.FederatedPeg.Tests.Wallet
+{
+    public class MultiSigTransactionsTests
+    {
+        private class TestData : IEnumerable<object[]>
```

Comment:
Lol, quick comment on what this is doing would be nice

Reply:
```       /// <summary>
        /// This class created all possible <c>true</c>/<c>false</c> combinations for input to <see cref="TransactionDataAddedToMultiSigTransactionsExistsInExpectedLookups(bool, bool, bool, bool, bool, bool)"/>.
        /// </summary>
```